test/cluster/mzworkflows.py

```diff
@@ -24,20 +24,17 @@
         name="dataflowd_1",
         options="--workers 2 --processes 2 --process 0 --hosts dataflowd_1:2101 dataflowd_2:2101",
         hostname="dataflowd_1",
-        depends_on=["dataflowd_2"],
```

Comment:
Those dependencies were not enforced in any way. In particular, this mzworkflows.py file created a circular dependency and docker-compose did not seem to care.